src/utils/url.ts

```diff
@@ -149,3 +149,16 @@ export const getLastUrlParam = (url: string) => {
 export const getUrlParamFromString = (data: string) => {
   return data.replaceAll(" ", "").toLowerCase();
 };
+
+export const resolveUrlPath: (path: string) => string = (path) => {
+  const base = window.location.href;
+  try {
+    return new URL(path, base).href;
+  } catch (error) {
+    Bugsnag.notify(new Error(error), (event) => {
+      event.severity = "info";
+      event.addMetadata("utils::url::resolveUrlPath", { path, base });
```

Comment:
```suggestion
      event.addMetadata("utils/url::resolveUrlPath", { path, base });
```

Reply:
We separate the folder/file with a `/` and then `::` for the function.

Reply:
This should go in the code style MD documents as my code is a copy from above in the same file